src/MongoDB.Driver.Core/Core/WireProtocol/CommandUsingQueryMessageWireProtocol.cs

```diff
@@ -79,6 +80,12 @@ internal class CommandUsingQueryMessageWireProtocol<TCommandResult> : IWireProto
             _resultSerializer = Ensure.IsNotNull(resultSerializer, nameof(resultSerializer));
             _messageEncoderSettings = messageEncoderSettings;
             _postWriteAction = postWriteAction; // can be null
+
+            if (messageEncoderSettings != null)
+            {
+                // BinaryDocumentFieldDecryptor is not required here
+                _documentFieldEncryptor = messageEncoderSettings.GetOrDefault<IBinaryDocumentFieldEncryptor>(MessageEncoderSettingsName.BinaryDocumentFieldEncryptor, null);
+            }
```

Comment:
Why would we ever do auto encryption/decryption using `OP_QUERY`.

I would have expected no changes to this class.